python/tvm/autotvm/task/dispatcher.py

```diff
@@ -178,6 +178,50 @@ def update(self, target, workload, cfg):
         self._config = cfg
 
 
+class ApplyFixedConfig(DispatchContext):
+    """Apply a config of a deterministic schedule.
+
+    Parameters
+    ----------
+    tasks : list[tvm.autotvm.task.task.Task]
+        List of autoTVM tasks.
+    schedule_name : str
+        Name of schedule to use.
+    """
+
+    def __init__(self, tasks, schedule_name: str):
```

Comment:
i think this would get called for each function being scheduled. so if the Relay program was more complex (e.g. task extraction found multiple tasks), then we'd want to specify a schedule for each.